src/components/NFTPage/PreviewCard.vue

```diff
@@ -96,6 +97,26 @@
         </template>
       </ButtonV2>
 
+      <template v-if="isContentViewable">
+        <ButtonV2
+          v-for="contentUrl in contentUrls"
+          :key="contentUrl"
+          class="mt-[12px]"
```

Comment:
Suggest adding `w-full` class for view buttons
```suggestion
          class="mt-[12px] w-full"
```